pkg/webhook/operatingsystemconfig/testdata/configure-rsyslog.sh

```diff
@@ -13,17 +13,40 @@ function configure_auditd() {
     mv /etc/audit/rules.d /etc/audit/rules.d.original
   fi
 
+  restart_auditd=false
+
   if [[ ! -d /etc/audit/rules.d ]]; then
     mkdir -p /etc/audit/rules.d
   fi
   if ! diff -rq /var/lib/rsyslog-relp-configurator/audit/rules.d /etc/audit/rules.d ; then
     rm -rf /etc/audit/rules.d/*
     cp -L /var/lib/rsyslog-relp-configurator/audit/rules.d/* /etc/audit/rules.d/
-    if [[ -f /etc/audit/plugins.d/syslog.conf ]]; then
-      sed -i 's/no/yes/g' /etc/audit/plugins.d/syslog.conf
-    fi
     augenrules --load
-    systemctl restart auditd
+    restart_auditd=true
+  fi
+
+  if [[ -f /etc/audit/plugins.d/syslog.conf ]] && grep -Fxq "active = no" "/etc/audit/plugins.d/syslog.conf" ; then
+    sed -i 's/no/yes/g' /etc/audit/plugins.d/syslog.conf
```

Comment:
```suggestion
    sed -i 's/active = no/active = yes/' /etc/audit/plugins.d/syslog.conf
```
I think this could have unexpected behaviour if the the file has `<another_field> = no`. Also the g option should not be necessary. Also if it is expected for the user to be able to potentially edit this file do we have the guarantee that there would be spaces between "active" and "no"(ex `active=no`)?

Reply:
That is the only option containing "no" or "yes" as per the specification. However, this could change in the future so I'll adapt the script.